vmware/ngc/NGC-Plugin/html/src/main/locale/en_US/org_opensds_storage_devices.properties

```diff
@@ -0,0 +1,132 @@
+summary.manageLink = Manage View
+nameLabel = Name:
+serverTypeLabel = Server Type:
+dimensionsLabel = Dimensions:
+
+# DeviceListView
+name = Name
+model = Type
+status = Status
+sn = SN
+ip = IP Address
+
+# Storage System Info
+system.label.ip = IP Address
+system.label.DeviceStatus = Status
+system.label.Location = Location
+system.label.ProductVersion = Version
+system.label.SN = SN
+system.label.WWN = WWN
+
+# Storage Alarms
+EventLevelE.INFO = Info
+EventLevelE.WARNING = Warning
+EventLevelE.MAJOR = Major
+EventLevelE.CRITICAL = Critical
+
+# Chassis Actions
+chassis.deleteAction = Remove Storage Platform
+chassis.createAction = Add Storage Platform
+chassis.configAction = Config Storage Platform
+
+# Other views
+monitor.view1 = Current Alarms
+monitor.view2 = Chassis Monitor2
+manage.view1 = Storage Pool
+manage.view2 = Chassis Manage2
+
+# Edit/Create dialog
+edit.header = <img src="/vsphere-client/opensds/assets/images/editChassisIcon.png"/> Update the device information
+create.header = <img src="/vsphere-client/opensds/assets/images/addChassisIcon.png"/> Enter the new device information
+create.submit = OK
+cancel = Cancel
+
+################### discovery start ##############
+storage.plugin.discovery.message.failed.hasdiscoveried = Device discovery failed. <br />The device has been discovered.
+storage.plugin.discovery.message.failed.hasSameDevice = Device discovery failed. <br />{0} and {1} refer to the same device, and {1} has been <br />discovered.
+storage.plugin.discovery.message.failed.readonly = Device discovery failed. <br />Read-only users have no permission to discover devices.
+storage.plugin.discovery.message.failed.1077949061 = Device discovery failed. <br />Incorrect username or password.<br />Discover again after verifying that <br />the username and password are correct.
+storage.plugin.discovery.message.failed.1077987870 = Device discovery failed. <br />Incorrect username or password.<br />Discover again after verifying that <br />the username and password are correct.
+storage.plugin.discovery.message.failed.1077949070 = Device discovery failed. <br />The user account has been locked. Please retry later.
+storage.plugin.discovery.message.failed.1077949071 = Device discovery failed. <br />The IP address has been locked.<br />Please add the vSphere web client IP <br />address into Authorized IP Addresses <br />Table in the storage device.
+storage.plugin.discovery.message.failed.1077987871 = Device discovery failed. <br />The user account has been locked. Please retry later.
+storage.plugin.discovery.message.failed.1077949058 = Device discovery failed. <br />The user does not have the permission.
+storage.plugin.discovery.message.failed.2000000009 = Device discovery failed. <br />The device IP address is incorrect or the network connection is abnormal. Please troubleshoot and retry.
+storage.plugin.discovery.message.failed.1077949067 = Device discovery failed. <br />The user quantity has reached the upper limit.
```

Comment:
Can we reword it as: It has exceeded the number of users limit.

Reply:
Thanks, we should reword it.

Reply:
Thanks

Reply:
Done